core/src/main/scala/kafka/tools/PerfConfig.scala

```diff
@@ -39,6 +39,8 @@ class PerfConfig(args: Array[String]) {
     .defaultsTo("yyyy-MM-dd HH:mm:ss:SSS")
   val showDetailedStatsOpt = parser.accepts("show-detailed-stats", "If set, stats are reported for each reporting " +
     "interval as configured by reporting-interval")
+  val showAllStatsOpt = parser.accepts("show-all-stats", "Show intermediary stats based on the " +
```

Comment:
I agree with @ijuma . It seems it's simpler to just always print the final result instead of introducing a new option. This is a change of behavior, but it probably matches most people's expectation. ProducerPerformance is already like that.